src/backend/controllers/AuthController.js

```diff
@@ -102,3 +102,38 @@ export const loginHandler = function (schema, request) {
     );
   }
 };
+
+/**
+ * This handler handles user verification.
+ * send POST Request at /api/auth/verify
+ * body contains {encodedToken}
+ * */
+
+export const verifyUser = function (schema, request) {
+  const { encodedToken } = JSON.parse(request.requestBody);
+  const decodedToken = jwt.verify(
+    encodedToken,
+    process.env.REACT_APP_JWT_SECRET
+  );
+  try {
+    if (decodedToken) {
+      const user = this.db.users.findBy({ email: decodedToken.email });
+      if (user) {
+        return new Response(200, {}, { user });
```

Comment:
You should handle the case if user doesn't exist

Reply:
for this token is invalid already there